Feat: Added a new cron-clone.sh script and edited the README.md file (#60)
stfc/cloud-docker-images: cloud-chatops/README.md

  "GITHUB_TOKEN": "<your-token>"
}
```

### Automatic Container Update

> To pull the latest `docker-compose` file automatically, you can copy the contents of the cron_clone.sh file and create a new file in either the cron.hourly, cron.daily, cron.weekly, or cron.monthly directories. These directories can be found in `$HOME/etc/...`.

Reviewer: ```suggestion
To pull the latest `docker-compose` file automatically, you can copy the cron_clone.sh file into any of the `cron.< hourly | daily | weekly | monthly > directories found in `/etc`.

Once created, you will need to change the file permissions accordingly:

```shell
  #Changing the file permissions.
  chmod 777 /etc/cron.hourly/<filename>

> 

Author: Great suggestion, I tested it and cron.hourly still works with +x and it makes sense to only give execute permissions.

> ```shell
>   #Changing the file permissions.
>   chmod +x /etc/cron.hourly/<filename>
> 
>   #You can test that the script is running correctly by using this command.
>   run-parts /etc/cron.hourly
> 
>   #if you don't recieve any output, the script has not run.
>   ```

Reviewer: If you try changing permisions while the file is in `/etc` you may need to use sudo. We should change the permissions inside the cloned repo before we copy the file over. This means we can avoid using sudo. 
e.g.
```suggestion
```shell
  # Clone the repo
  git clone https://github.com/stfc/cloud-docker-images.git

  # Change into repo directory
  cd cloud-docker-images/cloud-chatops

  # Change permissions on script
  chmod +x cron-clone.sh

  # Copy script to etc.hourly
  sudo cp cron-clone.sh /etc/cron.hourly/update_chatops

  # You can test that the script is running correctly by using this command.
  run-parts /etc/cron.hourly

  #if you don't recieve any output, the script has not run.
  ```
```

Author: Many thanks! Agreed with your suggestion around avoiding `sudo`, this makes sense as well copying the file rather than copying the contents. Avoids mistakes and makes it easier for whoever is deploying it.

Do we need `git clone https://github.com/stfc/cloud-docker-images.git` in this part of the README? I think if anyone is trying to automate this docker image, they will probably already have the repo. If they clone it again, might it risk having a duplicate repo?

Reviewer: It's up to you. Like you say. They have probably already cloned the repo

> *Please note when naming the file within $HOME/etc/..., you cannot use file fullstops '.' or file extenstions. Only valid characters are allowed [a-zA-Z0-9_-]*

Reviewer: ```suggestion
*Please note when naming the file within /etc/cron.<timeframe> you cannot use file extenstions. Only valid characters are allowed [a-zA-Z0-9_-]*
```

Author: That reads much better, I can get that changed over to `/etc/cron.<timeframe>` as well as the rest

### Automatic Container Update

To pull the latest `docker-compose` file automatically, you can copy the `chatopscron` file into any of the `cron.< hourly | daily | weekly | monthly > directories found in `/etc`.

Once created, you will need to change the file permissions accordingly:

```shell
  # Change into repo directory
  cd cloud-docker-images/cloud-chatops

  # Change permissions on script
  chmod +x chatopscron

>   # Copy script to etc.<timeframe>

Reviewer: ```suggestion
  # Copy script to etc/cron.<timeframe>
```
